apps/explorer/src/ui/SenderRecipient.tsx

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { cva } from 'class-variance-authority';
+
+//TODO: Switch to using ButtonOrLink component after support for sui explorer links
+import Longtext from '~/components/longtext/Longtext';
+import { Heading } from '~/ui/Heading';
+import { Text } from '~/ui/Text';
+
+const isCoinTransfer = cva(
+    ['flex flex-col ml-6 gap-[15px] justify-center  relative'],
+    {
+        variants: {
+            singleTransferCoin: {
+                true: 'before:content-[url()] before:border-2 before:border-[#a0b6c3] before:overflow-y-hidden before:absolute before:h-[calc(55%)] before:w-[15px] before:border-r-[transparent] before:border-t-[transparent] before:mt-1 before:ml-[-16px] before:rounded-l before:border-dotted',
```

Comment:
I'm curious if there's a better way to implement this that doesn't depend on `before`. I imagine there probably is.

Reply:
why not `before`?